python/vision_explanation_methods/evaluation/pointing_game.py

```diff
@@ -41,7 +42,7 @@ def pointing_game(self,
                       imagelocation: str,
                       index: int,
                       threshold: float = .8,
```

Comment:
nit: add some validation to throw ValueError if threshold is not between 0 and 1 and num_masks is a negative value